### DISCUSSION TITLE
How to change svg icon of next, prev button when using Swiper Element (WebComponent)?

### DISCUSSION THREAD

**Discussion by jony1993** [nolimits4web/swiper#7047] [Q&A]
Hi there,

Is there any easy way of changing the svg icon when using the Swiper Element (Webcomponent)?

**Comment by Koreanderson** [comment]
@jony1993

Probably the easiest way is to build your own next and prev elements with your custom styles attached to them.

Then on the swiper element you can specify your custom next and prev elements with css selectors on the swiper container attributes like so:

```
<swiper-container
  navigation-next-el=".custom-next-button"
  navigation-prev-el=".custom-prev-button"
>
...
</swiper-container>

<div class="custom-prev-button"><svg>...</svg></div>
<div class="custom-prev-button"><svg>...</svg></div>
```

https://swiperjs.com/swiper-api#param-navigation
https://swiperjs.com/element#parameters-as-attributes

**Reply by james0r** [reply]
This should probably go in the docs for the element. The Swiper Element is cool but you see "As easy as..." on the homepage and then realize it really ain't that easy at all.

**Reply by vrndstudio** [reply]
agreed 100%!

**Reply by bestquangminh** [reply]
it works perfectly ! thank you

**Reply by polopolur** [reply]
And if you would still like to keep some styles? For example, the location of the buttons relative to the slider

**Reply by 2gacem7** [reply]
Hi, very important, don't change the name of the classes : '.swiper-button-next',
 and '.swiper-button-prev' when you do it.

**Comment by labristow** [comment]
@Koreanderson answer is great but I was using React so I had to do very little tweak.

1. Note that <swiper-container> is synonymous to <Swiper> in ReactJs.
2. Instead of directly changing the default navigation class name inside <swiper-container> component props, you will need to do so inside the navigation property.

const sliderOptions = {
        navigation: {
            nextEl: ".nextButton",
            prevEl: ".prevButton",
    },
}
return (
    <React.Fragment>
        <button className="prevButton flex-center flex-shrink-0 w-11 h-11 rounded-full border border-gray-300">
                <SVGS.ArrowLeftIcon />
         </button>
        <Swiper {...sliderOptions}>
                 // SwiperSlides elements
        </Swiper>
        <button className="nextButton flex-center flex-shrink-0 w-11 h-11 rounded-full border border-gray-300">
                <SVGS.ArrowRightIcon />
         </button>
    </React.Fragment>
)
